cfgtool

```diff
@@ -104,7 +104,7 @@ vpn_pki_setup() {
   # connections.
   # This takes a long time but greatly enhances security. It should be repeated
   # every couple of moths or so.
-  # There is a standard set of 1024bit DH parameters, but attacks are known
+  # There is a standard set of 1024bit DH parameters, but attacks are known 
```

Comment:
This is a no-op change (just adds a space at the end). Please revert. It is a bit confusing for reviewers :)